do_like_javac/tools/wpi.py

```diff
@@ -141,18 +141,12 @@ def run(args, javac_commands, jars):
 
         while diffResult != 0:
 
-            iterationStubs = ':'.join(stubDirs)
-            stubArg = None
-
+            iterationAjavaDirs = ':'.join(ajavaDirs)
+            iterationCheckerCmd = checker_command
             if args.stubs:
```

Comment:
I'm concerned that this change isn't sufficient. There is a reason that `dljc` has a separate `stubs` command-line argument, rather than just allowing the user to pass any desired stub files along with the rest of the CF options as part of the `extraJavacArgs` argument: this code combined the stub files that the user started with (i.e. those in the command-line argument to `dljc`) with the generated stubs to pass just one `-Astubs` option to the Checker Framework.

With the switch to ajava files, I have two concerns:
* there will be an analogous problem for the `-Aajava` argument to the CF, if the user wants to run WPI with a predefined set of ajava files on all runs. `dljc` will therefore need an `ajava` argument.
* the `stubs` argument to `dljc` is now redundant. We might want to keep it so that old scripts continue to work, though, so I don't expect you to make that change in this PR, regardless.

Reply:
As a followup, my concern about the need for an `ajava` argument to `dljc` is moot if the CF supports multiple `-Aajava` command-line arguments and correctly combines them together. I don't know if it does that for stubs or not now, but I think it didn't when this code was originally written a few years back.

Reply:
@jwaataja Could you address this question?

Reply:
After testing, it seems like `SourceChecker.getOption()` returns the last argument to an option if the option was provided multiple times, so I'll add a new `ajava` option.

I'm still not sure it's necessary though. Currently, ajava files are mostly only useful as WPI output. I was under the impression the `stubs` option was so the user could provide stub files for annotated libraries, and they are probably still the best option for that use case.